keyboards/jacky_studio/piggy60/config.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 The QMK Community (@qmk)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define LED_PIN_ON_STATE 0
```

Comment:
Also, just a heads up, you can use "on_state" as true/false under "indicators" in the json file to set this.

Reply:
Ah, no worries, then.  

And if it helps, develop is scheduled to be merged into master at the end of the month (in 2 weeks).  Also, qmk.tzarc.io compiles firmware images from develop.